<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>&amp;排序：</string>
        </property>
        <property name="buddy">
         <cstring>comboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>85</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>按修改时间</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>按创建时间</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editSearch">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>搜索(Ctrl+F)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="CodeEditor" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>630</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="tabChangesFocus">
        <bool>false</bool>
       </property>
       <property name="tabStopWidth">
        <number>40</number>
       </property>
      </widget>
      <widget class="QWebView" name="webView">
       <property name="font">
        <font>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::PreventContextMenu</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="locale">
        <locale language="Chinese" country="China"/>
       </property>
       <property name="url">
        <url>
         <string>about:blank</string>
        </url>
       </property>
       <property name="renderHints">
        <set>QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Close"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Find"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="action_Help"/>
   </widget>
   <widget class="QMenu" name="menu_G">
    <property name="title">
     <string>格式(&amp;G)</string>
    </property>
    <addaction name="action_FirstTitle"/>
    <addaction name="action_SencondTitle"/>
    <addaction name="action_ThirdTitle"/>
    <addaction name="action_ForthTitle"/>
    <addaction name="action_Bold"/>
    <addaction name="action_Italic"/>
    <addaction name="action_Hrline"/>
    <addaction name="action_String"/>
    <addaction name="action_Quote"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(&amp;G)</string>
    </property>
    <addaction name="action_EditMode"/>
    <addaction name="action_ViewMode"/>
    <addaction name="actionFormat"/>
    <addaction name="actionDouble"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_G"/>
   <addaction name="menu_V"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Paste"/>
   <addaction name="separator"/>
   <addaction name="action_Find"/>
   <addaction name="separator"/>
   <addaction name="action_EditMode"/>
   <addaction name="action_ViewMode"/>
   <addaction name="actionFormat"/>
   <addaction name="actionDouble"/>
   <addaction name="action_Exit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="formatToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Italic"/>
   <addaction name="action_Bold"/>
   <addaction name="action_FirstTitle"/>
   <addaction name="action_SencondTitle"/>
   <addaction name="action_ForthTitle"/>
   <addaction name="action_Hrline"/>
   <addaction name="action_String"/>
   <addaction name="action_Quote"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/document-new.png</normaloff>:/myImages/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/document-open.png</normaloff>:/myImages/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/gtk-close.png</normaloff>:/myImages/images/gtk-close.png</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/document-save.png</normaloff>:/myImages/images/document-save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="action_SaveAs">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/document-save-as.png</normaloff>:/myImages/images/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/exit.png</normaloff>:/myImages/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/edit-undo.png</normaloff>:/myImages/images/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/editcut.png</normaloff>:/myImages/images/editcut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/editcopy.png</normaloff>:/myImages/images/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/editpaste.png</normaloff>:/myImages/images/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/edit-find.png</normaloff>:/myImages/images/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/help.png</normaloff>:/myImages/images/help.png</iconset>
   </property>
   <property name="text">
    <string>版本说明(&amp;H)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Italic">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/textitalic.png</normaloff>:/myImages/images/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>斜体(&amp;I)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Bold">
   <property name="icon">
    <iconset resource="myResource.qrc">
     <normaloff>:/myImages/images/textbold.png</normaloff>:/myImages/images/textbold.png</iconset>
   </property>
   <property name="text">
    <string>粗体(&amp;B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_FirstTitle">
   <property name="text">
    <string>一级标题</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="action_SencondTitle">
   <property name="text">
    <string>二级标题</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="action_ThirdTitle">
   <property name="text">
    <string>三级标题</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="action_ForthTitle">
   <property name="text">
    <string>四级标题</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="action_Hrline">
   <property name="text">
    <string>水平线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_String">
   <property name="text">
    <string>字符串</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quote">
   <property name="text">
    <string>引用</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_EditMode">
   <property name="text">
    <string>编辑模式</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="action_ViewMode">
   <property name="text">
    <string>查看模式</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFormat">
   <property name="text">
    <string>format</string>
   </property>
  </action>
  <action name="actionDouble">
   <property name="text">
    <string>双屏</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myResource.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_lineEdit_textChanged(QString)</slot>
  <slot>on_pbCancel_clicked()</slot>
  <slot>on_pbSearch_clicked()</slot>
  <slot>on_cbCaseSenstive_toggled(bool)</slot>
  <slot>on_editText_textChanged()</slot>
 </slots>
</ui>
